fec.config.js

```diff
@@ -0,0 +1,37 @@
+const { resolve } = require('path');
+
+module.exports = {
+  appUrl: '/insights/advisor',
+  debug: true,
+  useProxy: process.env.PROXY === 'true',
+  proxyVerbose: true,
+  /**
+   * Change to false after your app is registered in configuration files
+   */
+  sassPrefix: 'advisor, inventory',
+  interceptChromeConfig: false,
+  /**
+   * Add additional webpack plugins
+   */
+  plugins: [],
+  ...(process.env.HOT
+    ? { hotReload: process.env.HOT === 'true' }
+    : { hotReload: true }),
+  ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
+  moduleFederation: {
```

Comment:
We should keep this in line with the other apps and not have `exclude`. See https://github.com/RedHatInsights/patchman-ui/blob/master/fec.config.js#L12-L21

Reply:
Thanks I updated it